add-ons/aws-load-balancer-controller/Chart.yaml

```diff
@@ -5,13 +5,13 @@ type: application
 
 # The chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.1.2
+version: 0.1.3
 
 # Version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
 appVersion: "1.0"
 
 dependencies:
 - name: aws-load-balancer-controller
-  version: 1.4.7
+  version: 2.8.1
```

Comment:
This is not the correct version. This version points to the chart version and not the controller version. You want version `1.8.1`

Reply:
Wrong value updated.